infra/nginx.conf

```diff
@@ -1,6 +1,7 @@
 server_tokens off;
 
 server {
+        listen 80;
         listen 443 ssl;
 
         server_name www.grtp.co;
```

Comment:
If I'm reading the [docs](http://nginx.org/en/docs/http/request_processing.html) correctly, `http://grtp.co/` will match this `server` block because it's the only one listening on port 80.